graphrag_sdk/custom_metric.py

```
@@ -0,0 +1,548 @@
+    construct_verbose_logs,
+    trimAndLoadJson,
+    check_llm_test_case_params,
+    initialize_model,
+)
+from deepeval.test_case import (
+    LLMTestCase,
+    LLMTestCaseParams,
+    ConversationalTestCase,
+)
+
+from deepeval.metrics.contextual_relevancy.schema import *
```

Ruff (0.8.2)

`from deepeval.metrics.contextual_relevancy.schema import *` used; unable to detect undefined names

(F403)